test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/MiscCryptoSuite.java

```diff
@@ -56,7 +57,8 @@ public static void main(String... args) {
 	private List<HapiApiSpec> positiveTests() {
 		return Arrays.asList(
 //				transferChangesBalance()
-				getsGenesisBalance()
+//				getsGenesisBalance()
```

Comment:
Do you intend to comment out this test?

Reply:
It seems to me that this suite serves as a playground for manual and convenient testing, for example, to get Genesis balance of a network, which we don't always want run that as a functionality test. That was why I picked this suite to add the `reduceTransferFee` test to show that the fee reduction works. Again, `reduceTransferFee` is not necessarily tested for functionality.

Reply:
Got it.